src/pages/research.astro

```diff
@@ -0,0 +1,1170 @@
+                              class="btn-action flex items-center"
+                              aria-label="Download preprint"
+                            >
+                              <span class="sr-only">Download preprint</span>
+                              <svg
+                                xmlns="http://www.w3.org/2000/svg"
+                                class="h-5 w-5"
+                                viewBox="0 0 20 20"
+                                fill="currentColor"
+                              >
+                                <path
+                                  fill-rule="evenodd"
+                                  d="M6 2a2 2 0 00-2 2v12a2 2 0 002 2h8a2 2 0 002-2V7.414A2 2 0 0015.414 6L12 2.586A2 2 0 0010.586 2H6zm5 6a1 1 0 10-2 0v3.586l-1.293-1.293a1 1 0 10-1.414 1.414l3 3a1 1 0 001.414 0l3-3a1 1 0 00-1.414-1.414L11 11.586V8z"
+                                  clip-rule="evenodd"
+                                />
+                              </svg>
+                              <span class="ml-2 text-sm font-bold text-gray-400 move-button">
+                                PREPRINT
+                              </span>
+                            </a>
+                          )}
+                          {paper.doiOrPublisherUrl && (
+                            <a
+                              href={
+                                paper.doiOrPublisherUrl.startsWith("10.")
+                                  ? `https://doi.org/${paper.doiOrPublisherUrl}`
+                                  : paper.doiOrPublisherUrl
+                              }
+                              class="btn-action flex items-center"
+                              aria-label="View publisher version"
+                            >
+                              <span class="sr-only">
+                                View publisher version
+                              </span>
+                              <svg
+                                xmlns="http://www.w3.org/2000/svg"
+                                class="h-5 w-5"
+                                viewBox="0 0 20 20"
+                                fill="currentColor"
+                              >
+                                <path d="M11 3a1 1 0 100 2h2.586l-6.293 6.293a1 1 0 101.414 1.414L15 6.414V9a1 1 0 102 0V4a1 1 0 00-1-1h-5z" />
+                                <path d="M5 5a2 2 0 00-2 2v8a2 2 0 002 2h8a2 2 0 002-2v-3a1 1 0 10-2 0v3H5V7h3a1 1 0 000-2H5z" />
+                              </svg>
+                              <span class="ml-2 text-sm font-bold text-gray-400 move-button">
+                                DOI
+                              </span>
+                            </a>
+                          )}
+                        </div>
+                        <div class="ml-2">
+                          {paper.abstract && (
+                            <button
```

Comment:
There's no reason whatsoever for this to be javascript-only. It could easily be a `<details>` element with the appropriate styling. Not a blocker, but I'd strongly prever to (eventually) not have to use any scripting

Reply:
It does not work with `<details>` or at least I cannot make it work without significantly changing the overall layout and compromising the design. I made it work using a checkbox now similar to this: https://stackoverflow.com/questions/41220717/collapse-without-javascript/41220875#41220875.